src/.nuget/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="NuGet.CommandLine" version="2.8.2" />
```

Comment:
This is not a deliberate change.  Something got automatically altered without me telling it to here.